.code-samples.meilisearch.yaml

```diff
@@ -401,15 +401,15 @@ getting_started_add_documents_md: |-
   <dependency>
     <groupId>com.meilisearch.sdk</groupId>
     <artifactId>meilisearch-java</artifactId>
-    <version>0.7.2</version>
```

Comment:
I also think you can add to the changelog that you removed some classes like `GenericServiceTemplate`, `Dump`, `DumpHandler`, `MeilisearchHttpRequest`, and others...  

The users will search your CHANGELOG for keywords like "why the class I was using does not exist anymore?". "Now this is the new way to use the methodABC" and so on (like you did for the exception classes)...